docs/release/trg-5/trg-5-05.md

```diff
@@ -36,14 +39,66 @@ image:
   ## pullSecrets:
   ##   - myRegistryKeySecretName
   ##
-  pullSecrets: [ ]
+  pullSecrets: []
   ## Set to true if you would like to see extra information on logs
   ##
 ```
 
 :::caution do not use `latest` as image tag!
 
 This might cause unexpected effects, as the Helm release could change without bumping the chart version! Also due to
-security reasons you shall not use `latest` tag.
+security reasons you shall not use `latest` tag. If the image tag should be set to a pinned tag (e.g.: `latest`) anyway, it should done in an additional (environment specific) values.yaml and in this case the image's **pullPolicy** is recommended to be set to `Always`.
 
 :::
+
+## Ingress
+
+- Ingress **must** be disabled by default
+- Hostname **must** not be set by default
+
+An example section for ingress resources:
+
+```yaml
+ingress:
+  enabled: false
+  # className: "nginx"
+  ## Optional annotations when using the nginx ingress class
+  #annotations:
+  #  nginx.ingress.kubernetes.io/rewrite-target: "/$1"
+  #  nginx.ingress.kubernetes.io/use-regex: "true"
+  #  nginx.ingress.kubernetes.io/enable-cors: "true"
+  #  nginx.ingress.kubernetes.io/cors-allow-origin: "https://*.example.org"
+  tls:
+    - secretName: ""
+      hosts:
+        - ""
+  hosts:
+    - host: ""
+      paths:
+        - path: "/"
+          pathType: "Prefix"
+          backend:
+            service: "app"
+            port: 8080
+```
+
+## Dependencies
+
+An example section for Postgresql dependency configuration:
+
+```yaml
+postgresql:
+  ## PostgreSQL chart configuration
+  enabled: true
+  ## Dependencies should be prefixed with the `nameOverride` property
+  nameOverride: "app-postgresql"
```

Comment:
Could you please expand on why setting a nameOverride is beneficial? I see the benefits when using the chart then in an umbrella scenario but why setting it in the product specific charts already I don't get. 
IMO the default naming behaviour of the Bitnami chart is sufficient and no nameOverride is needed.

Reply:
and sorry that I didn't spot this at my last review

Reply:
made some adjustments here